product-scenarios/4-single-sign-on/4.1-sso-for-web-app/4.1.4-sso-with-oidc/4.1.4.1-dcr/src/test/java/org/wso2/identity/scenarios/sso/test/dcr/DCRTestCase.java

```diff
@@ -98,7 +98,7 @@ private static Object[][] dcrConfigProvider() throws Exception {
                         getRegisterRequestJSON("request2.json"), getUpdateRequestJSON("request2.json"), ADMIN_USERNAME,
                         ADMIN_PASSWORD, SUPER_TENANT_DOMAIN
                 }
-        };
+                };
```

Comment:
Unreleated change

Reply:
fixed in https://github.com/wso2/product-is/pull/17047/commits/1cdda47d18ad6c1a03a59c44b735c38946dade4f

Reply:
seems not yet updated.